### ISSUE TITLE
Not reporting error when elm-test is unavailable

### ISSUE THREAD

**Issue by razzeee** [frawa/vscode-elm-test-runner#33]
I tried to use this with the spa example https://github.com/rtfeldman/elm-spa-example and it seems to just spin the circular arrow button forever.

```
[2021-04-16 22:31:57.199] [INFO] Test Explorer found
[2021-04-16 22:31:57.199] [INFO] Creating adapter for /home/razze/Development/elm-spa-example
[2021-04-16 22:31:57.199] [INFO] Initializing Elm Test Runner adapter
[2021-04-16 22:31:57.199] [INFO] Registering adapter for /home/razze/Development/elm-spa-example
[2021-04-16 22:31:57.200] [INFO] Running Elm Tests as task [ 'elm-test' ]
[2021-04-16 22:31:57.200] [INFO] Initialization finished
[2021-04-16 22:31:57.839] [INFO] Running Elm Tests [ 'elm-test' ]

```

Is there something I'm missing?

**Comment by razzeee** [comment]
Whoops, this seems to happen, if you don't have `elm-test` installed. So it would be nice, if it would error better.

**Comment by frawa** [comment]
Resolved in 0.9.7
